### ISSUE TITLE
Lists for college offices are different to the data showed to users 

### ISSUE THREAD

**Issue by jinbozz** [OpenJUB/jRooms-legacy#3]
jRoom has a huge bug right now, certain people didn't get the rooms they chose and confirmed. Such cases happen in all four colleges, quite a lot.

The data showed to users and the data printed as lists to college officer are clearly different. Typical behavior is that, someone chose a room, and eventually he/she was in "unallocated student list".

Please fix that as soon as possible.

**Comment by tkw1536** [comment]
@ungureanuvladvictor @dmitryfd  @filipbitola 

(@sourabhlal ?)

This should be fixed somehow.

**Comment by dmitrc** [comment]
Not everyone is getting their first choice room, as there is demand bigger than the supply. AFAIK, only instance of unallocated people we had was when they didn't choose college or roommate (for 2-3 people rooms). Otherwise, process was fully transparent and the data that was outputted as a result is a final list given to the colleges.

I will ask @filipbitola go investigate, but I suspect there is no bug here. Would you care to provide specific examples, otherwise?

**Comment by jinbozz** [comment]
I don't think it is problem of first choice or whatsoever. At least 2 problems:
1. rommie relationship is broken. No matter they get the rooms or not, they should be in the same situation. i,e: Either they live in adjacent rooms, or they both get unallocated. In this case, my roomie gets the right room, but I stay unallocated.
2. The data shown to the user and the data on the final lists are different. I got a email screenshot at that time: 

![unnamed qq screenshot20150819153302](https://cloud.githubusercontent.com/assets/10081485/9357537/ce3a7ebe-4687-11e5-88fb-59b78f21083c.png)

And after manual fixing, the problem is solved, only for the user side, NOT on the final lists.

If you need more details, I am willing to help you.

**Comment by filipbitola** [comment]
The screenshot was resolved quite a while ago, way before the end of the phases. There was no issue with the allocations at the time the final phase closed, otherwise we would have heard. Are you sure these final results are from then and not from after the server went offline or before the phases closed?

**Comment by jinbozz** [comment]
I am not sure. The girl got a confirmation at the time. Here is the screenshot she took:
![mmexport1439993091071](https://cloud.githubusercontent.com/assets/10081485/9358470/8f5468fe-468c-11e5-9bbd-d752a1963d51.jpg)

But this is the list Tanja(the college officer of Nordmetall) got:
![mmexport1439993001728](https://cloud.githubusercontent.com/assets/10081485/9358476/992d36d0-468c-11e5-8179-d1d136c8bcf0.jpg)

You can see she stays unallocated.

Right now the room NB-397 belongs to an exchange student from Netherlands.

**Comment by filipbitola** [comment]
![bullshit](https://cloud.githubusercontent.com/assets/2878348/9358879/9587e6f4-468e-11e5-84fa-cd502d4a3555.JPG)
Here's what's in the database at this very moment. Can you give me a complete list of all the people that have this issue?

**Comment by ungureanuvladvictor** [comment]
are we still maintaining this ?
